omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderUtil.java

```diff
@@ -22,6 +22,10 @@
 import org.openmrs.api.context.Context;
 
 public class OrderUtil {
+
+	private OrderUtil(){
+		throw new IllegalStateException("Utility class");
```

Comment:
Interesting idea, I didn't think of it myself. The build process runs without any warnings or errors? (unfortunately I can't compile it on the computer I'm working on at the moment)

Reply:
@sacull worked fine in my computer and i followed the resource provided by you https://sonar.openmrs.org/coding_rules#rule_key=squid%3AS1118